framework/interface/ui_handlers.py

```diff
@@ -440,11 +441,12 @@ class FileRedirectHandler(custom_handlers.UIRequestHandler):
     SUPPORTED_METHODS = ('GET')
 
     def get(self, file_url):
+	file_url = file_url[1:]
```

Comment:
It probably means there was a mixed indentation (tabs and spaces). Always use spaces. :)